schedule/functional/allpatterns.yaml

```diff
@@ -47,7 +47,6 @@ schedule:
     - console/glibc_sanity
     - console/zypper_in
     - console/zypper_log
-    - console/yast2_i
```

Comment:
This module is about software installation so I think we should cover this part this part deleted here as well: https://openqa.suse.de/tests/4308925# adding the module to this test suite, so we could have more chances to catch a bug, wdyt?

Reply:
i was checking this suite as well but for the FullMedia instead of Online. But i let it out of this PR first because of simplicity and second because according to the description indicates testing on _enabling all modules and selecting all patterns._ So i do not think that the yast2_i applies here but ofc we can do it. However the question for me is Online or Full media(as we have some coverage already for Online)

Reply:
In functional groups there was only one job and for that medium is still work in progress for both teams to add more tests there, so I would say that for Online at least we should cover that.